Networking/Networking/Model/StoreOnboardingTask.swift

```diff
@@ -60,11 +60,11 @@ private extension StoreOnboardingTask.TaskType {
             return 1
         case .addFirstProduct:
             return 2
-        case .launchStore:
+        case .payments, .woocommercePayments:
```

Comment:
Nit: Kindly consider adding a unit test to check the sort order.

Reply:
These are already tested in `StoreOnboardingViewModelTests` and failed - updated in 2d56e7b2e506f4928895c5f63cc684b7128b4197.